io_scene_godot/converters/material_node_tree/exporters.py

```diff
@@ -0,0 +1,121 @@
+"""Interface for node tree exporter"""
+import os
+from shutil import copyfile
+import bpy
+from .shaders import ShaderGlobals, FragmentShader, VertexShader
+from .node_vistors import find_node_visitor
+from ...structures import InternalResource, ExternalResource, ValidationError
+
+
+def find_material_output_node(node_tree):
+    """Find materia output node in the material node tree, if
+    two output nodes found, raise error"""
+    output_node = None
+    for node in node_tree.nodes:
+        if node.bl_idname == 'ShaderNodeOutputMaterial':
+            if output_node is None:
+                output_node = node
+            else:
+                raise ValidationError(
```

Comment:
Probably change to a logging.warning() and return one of the output nodes.

Reply:
Ah, yeah, it should be warning